spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/src/main/java/org/springframework/cloud/gcp/core/autoconfig/GCPContextAutoConfiguration.java

```diff
@@ -23,16 +22,14 @@
 @EnableConfigurationProperties(GCPProperties.class)
 public class GCPContextAutoConfiguration {
 
-
 	@Autowired
 	private GCPProperties gcpProperties;
 
 	@Bean
-	public GoogleCredentials googleCredentials(ResourceLoader resourceLoader) throws Exception {
-		if (!StringUtils.isEmpty(this.gcpProperties.getJsonKeyFile())) {
+	public GoogleCredentials googleCredentials() throws Exception {
+		if (!StringUtils.isEmpty(this.gcpProperties.getCredentialKeyFile())) {
 			return GoogleCredentials.fromStream(
-					resourceLoader.getResource(this.gcpProperties.getJsonKeyFile())
-							.getInputStream());
+					this.gcpProperties.getCredentialKeyFile().getInputStream());
```

Comment:
Are we going to have tests on the matter?
I don't exactly for this change, but the whole test suit for the `GCPContextAutoConfiguration` at least on mocks.

Just an idea, no need to address in this PR.
Thanks